app/components/Checkbox.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from 'react';
+import { Checkbox } from 'react-native-paper';
+
+const checkboxComponent = () => {
```

Comment:
rename the component to `Checkbox`. You can import the `Checkbox` from React Native Paper by doing
```ts
import { Checkbox as PaperCheckbox } from 'react-native-paper;
```
to allow you to name your component `Checkbox`.